v2v-helper/pkg/constants/constants.go

```diff
@@ -142,4 +145,7 @@ echo "$(date '+%Y-%m-%d %H:%M:%S') - Network fix script completed" >> "$LOG_FILE
 
 	// VMwareCredsRequeueAfterMinutes is the time to requeue after.
 	VMwareCredsRequeueAfterMinutes = 60
+
+	// ValidateRDMOwnerVMsKey is the key for enabling/disabling RDM owner VM validation
+	ValidateRDMOwnerVMsKey = "VALIDATE_RDM_OWNER_VMS"
```

Comment:
need to add this `VALIDATE_RDM_OWNER_VMS` to `image_builder/configs/vjailbreak-settings.yaml`